packages/safe-apps-developer-ui/src/components/pages/safes/apps/AppIframe.tsx

```diff
@@ -13,9 +20,16 @@ const SIframe = styled.iframe`
   height: 100%;
 `;
 
+type ProposedTxs = {
+  transactions: Transaction[];
+  requestId: RequestId;
+  params: SendTransactionParams;
```

Comment:
Are these params used somewhere?

Reply:
now they're passed to the tx creation func, good catch!